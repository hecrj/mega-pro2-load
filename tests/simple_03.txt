20

10
20
30
40
50
60
70
80
90
100
110
120
130
140
150
160
170
180
190
200

11
1 2 0 0 3 4 0 0 5 6 0 0 7 8 0 0 9 10 0 0 11 0 0

10
3 1 2 3

20
3 4 5 6

30
3 7 8 9

40
3 10 11 12

50
3 13 14 15

60
3 16 17 18

70
3 19 20 1

80
3 2 3 4

90
3 5 6 7

100
3 8 9 10

10
3 11 12 13

-3 1
-5 1 2

-1 13 100
-3 101

-1 20 102
-1 20 103

-3 104

-6