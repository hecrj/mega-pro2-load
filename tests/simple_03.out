Servidores ocupados
0
Pelicula mas solicitada
0 0
Peticion procesada y servidores
0 3
5 11
Servidores ocupados
5 0 2
11 0 2
Peticion procesada y servidores
1 3
7
Peticion procesada y servidores
2 0
Servidores ocupados
7 1 1
